db/migrate/20230821124053_create_tasks.rb

```diff
@@ -1,20 +1,19 @@
 class CreateTasks < ActiveRecord::Migration[7.0]
-  def change
+  def up
     create_table :tasks do |t|
-      t.integer "user_id", :null => false
-      t.string "title", :limit => 25
-      t.text "content", :limit => 300
+      t.integer "user_id",index: true, foreign_key: true, :null => false
+      t.string "title"
+      t.text "content"
       t.date "deadline"
       t.integer "importance"
       t.string "status", :default => "to do"
```

Comment:
lets introduce enums in a different PR and not here therefore lets remove the default and keep it a string only
```suggestion
      t.string "status"
```